app/answers/answer_store.py

```diff
@@ -25,29 +30,36 @@ def clear_answers(self):
 class AnswerStore(AbstractAnswerStore):
 
     def store_answer(self, key, value):
-        data = current_user.get_questionnaire_data()
-        if ANSWERS not in data:
-            answers = {key: value}
-            data[ANSWERS] = answers
-        else:
-            data[ANSWERS][key] = value
+        pass
 
     def get_answer(self, key):
-        data = current_user.get_questionnaire_data()
-        if ANSWERS not in data.keys():
-            data[ANSWERS] = {}
-            return None
-        if key not in data[ANSWERS].keys():
+        # ujm = UserJourneyManager.get_instance()
+        # current_state = ujm.get_current_state()
+        # answer = current_state.get_answer(key)
+        # return answer.input
+        #
+        answers = self.get_answers()
```

Comment:
Why not have answers as a property of an AnswerStore? That way this code code be refined (even removed as you could then call `answerStoreInstance.answers[key]` which assumes that .answers has a dict like interface but would simplify this code and other instances of it.